index.html

```diff
@@ -63,86 +70,100 @@ <h1>Eat Donuts in the Wilderness</h1>
         </div>
```

Comment:
On Line 67 `<content class="h1-container">` could be updated to: `<container class="h1-container d-flex justify-content-center align-items-center">` and then in the css line 183 `.h1-container` could be updated to `.h1-container{ 
  height: 150px;
  background-color: var(--background);
}`